src/sentry/integrations/vsts/integration.py

```diff
@@ -69,6 +70,18 @@ def get_repositories(self):
             })
         return data
 
+    def get_link_issue_config(self, group, **kwargs):
```

Comment:
why do you need to redefine this here? aren't you just overriding what you have in issues.py?

Reply:
you're right.thank you good catch.